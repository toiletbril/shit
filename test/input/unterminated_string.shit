"This is a string. But where is another quote?