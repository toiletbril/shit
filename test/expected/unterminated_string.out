shit: 1:1: Error: Unterminated string literal.
     1 |  "This is a string. But where is another quote?
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ here.
shit: 1:47: Note:
     1 |  "This is a string. But where is another quote?
                                                        ^~ expected a " here.
